src/controllers/securityQuestionController.ts

```diff
@@ -0,0 +1,82 @@
+import { SecurityQuestion } from '../models/';
+import { Request, Response, NextFunction } from 'express';
+import { NotFoundError } from '../errors/httpErrors';
+
+async function retrieveSecurityQuestion(req: Request, res: Response, next: NextFunction) {
+  try {
+    const securityQuestion = await SecurityQuestion.findByPk(req.params.id);
+    if (securityQuestion === null) {
+      throw new NotFoundError('Security Question cannot be found');
+    }
+    req.securityQuestion = securityQuestion;
+    next();
+  } catch (err) {
+    next(err);
+  }
+}
+
+async function indexSecurityQuestion(req: Request, res: Response, next: NextFunction) {
```

Comment:
I think it would be good to switch the default route to return only the `isActive` questions. Have an explicitly route for getting all the security questions because that route is rarely used.

Reply:
This is so frontend does not need to be aware that we have a `isActive` implementation. 
E.g.
to get `isActive` -> `/securityQuestions`
to get all -> `/securityQuestions/all`

Reply:
Ok switched and updated docs